src/backend/Auth/Auth.csproj

```diff
@@ -9,6 +9,10 @@
 		<Product>In good hands</Product>
 	</PropertyGroup>
 
+	<ItemGroup>
+	  <PackageReference Include="Z.EntityFramework.Plus.EFCore" Version="7.18.5" />
```

Comment:
Skoro jest już dodane w `Core.csproj` to tu trzeba wywalic, redundancja.

Reply:
Usunięte